kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/AddFilter.tsx

```diff
@@ -43,6 +44,19 @@ const AddFilter: React.FC<FilterModalProps> = ({
 
   const onSubmit = React.useCallback(
     async (values: AddMessageFilters) => {
+      const isFilterExists = filters.some(
+        (filter) => filter.name === values.name
+      );
+
+      if (isFilterExists) {
+        showAlert('error', {
+          id: '',
+          title: 'Validation Error',
+          message: 'Filter with the current name already exists',
```

Comment:
i would suggest to use `same` instead of current here 
```suggestion
          message: 'Filter with the same name already exists',
```

Reply:
Updated